File: Frontend/src/types/OcrResponseDTO.ts
export interface OcrItemDTO {
    name: string;
    quantity?: number;
    unitPrice?: number;
}

export interface OcrResponseDTO {
    amount: number;
    category: string;
    date: string;
    locationName?: string;
    confidence: number;
    receiptUrl?: string;
    items?: OcrItemDTO[];
}

File: backend/src/main/java/com/familie/cheltuieli_familie/dto/OcrResponseDTO.java
package com.familie.cheltuieli_familie.dto;

import java.math.BigDecimal;
import java.util.List;

public record OcrResponseDTO(
    BigDecimal amount,
    String category,
    String date,
    String locationName,
    double confidence,
    List<OcrItemDTO> items
) {}

